src/components/PlayerView/index.tsx

```diff
@@ -103,6 +103,10 @@ export function PlayerView({
         );
       }
     }
+    return () => {
+      fullscreenBridge.current?.destroy();
+      fullscreenBridge.current = undefined;
+    };
```

Comment:
This was added to clean up fullscreen bridge once the player view gets deallocated.